ansible/sync_jobs.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+# Created by argbash-init v2.6.1
+# ARG_POSITIONAL_SINGLE([host],[The Jenkins host entry as you have in your ssh configuration],[jenkins])
+# ARG_OPTIONAL_REPEATED([job],[j],[Jobs to synchronize, pass none to auto-detect and synchronize all])
+# ARG_OPTIONAL_BOOLEAN([verbose],[v])
+# ARG_HELP([Synchronize jobs configurations ])
+# ARG_DEFAULTS_POS()
+# DEFINE_SCRIPT_DIR()
+# ARGBASH_GO()
+# needed because of Argbash --> m4_ignore([
+### START OF CODE GENERATED BY Argbash v2.6.1 one line above ###
+# Argbash is a bash code generator used to get arguments parsing right.
+# Argbash is FREE SOFTWARE, see https://argbash.io for more info
+
+die()
+{
+	local _ret=$2
+	test -n "$_ret" || _ret=1
+	test "$_PRINT_HELP" = yes && print_help >&2
+	echo "$1" >&2
+	exit ${_ret}
+}
+
+begins_with_short_option()
+{
+	local first_option all_short_options
+	all_short_options='jvh'
+	first_option="${1:0:1}"
+	test "$all_short_options" = "${all_short_options/$first_option/}" && return 1 || return 0
+}
+
+
+
+# THE DEFAULTS INITIALIZATION - POSITIONALS
+_positionals=()
+_arg_host="jenkins"
+# THE DEFAULTS INITIALIZATION - OPTIONALS
+_arg_job=()
+_arg_verbose="off"
+
+print_help ()
+{
+	printf '%s\n' "Synchronize jobs configurations "
```

Comment:
To `Synchronize` gives the impression that the jobs configurations go both ways, while the tool only downloads from the server to local repo.

Reply:
Yes, I have rebranded that to `pull`.